apps/admin-gui/src/app/shared/components/add-owner-dialog/add-owner-dialog.component.html

```diff
@@ -0,0 +1,56 @@
+<div class="user-theme">
+  <h1 mat-dialog-title>{{"DIALOGS.ADD_OWNER.TITLE" | translate}}</h1>
+  <mat-spinner class="ml-auto mr-auto" *ngIf="loading"></mat-spinner>
+  <div *ngIf="!loading">
+    <div mat-dialog-content class="dialog-container">
+      <mat-form-field>
+        <input
+          matInput
+          required
+          placeholder="{{'DIALOGS.ADD_OWNER.NAME' | translate}}"
+          [formControl]="nameCtrl">
+        <mat-error>
+          {{'DIALOGS.ADD_OWNER.FILL_NAME' | translate}}
+        </mat-error>
+      </mat-form-field>
+      <mat-form-field>
+        <input
+          matInput
+          required
+          placeholder="{{'DIALOGS.ADD_OWNER.CONTACT' | translate}}"
+          [formControl]="contactCtrl">
+        <mat-error *ngIf="contactCtrl.errors.required">
+          {{'DIALOGS.ADD_OWNER.FILL_CONTACT' | translate}}
+        </mat-error>
+        <mat-error *ngIf="contactCtrl.invalid && !contactCtrl.errors.required">
+          {{'DIALOGS.ADD_OWNER.CONTACT_FORMAT' | translate}}
+        </mat-error>
+      </mat-form-field>
+      <mat-form-field appearance="fill">
```

Comment:
```suggestion
      <mat-form-field>
```
usually we have this style for mat-select